README.md

```diff
@@ -138,3 +158,99 @@ HTTP Response:
     "count": int
 }
 ```
+
+## Adding Alarm Metric Log
+
+POST "/metrics/alarm"
+
+Request Body:
+
+```
+{
+    "alarmInfo": [
+        {
+            "username": string,
+            "startTime": string, // get UTC string with: new Date().toUTCString()
+            "endTime": string // get UTC string with: new Date().toUTCString()
+        }
+        ...
+    ]
+}
+```
+
+HTTP Response:
+
+```
+{
+    "addedAlarm": [
+        {
+            "alarmID": integer // keep returned id for future log updates
+        }
+        ...
+    ]
+}
+```
+
+## Update Alarm Log Sent Status
+
+PUT "/metrics/alarm"
+
+Request Body:
+
+```
+{
+    "alarmInfo": [
+        {
+            "alarmID": integer,
+            "sentStatus": boolean
+        }
+        ...
+    ]
+}
+```
+
+HTTP Response:
+
+```
+{
+    "response": [
+        {
+            "success": boolean,
+            "message": string
+        }
+        ...
+    ]
+}
+```
+
+## Update Alarm Log End Time
+
+PUT "/metrics/alarm/extend"
```

Comment:
This should be the same PUT endpoint to "Update Alarm Log Sent Status". The way you can detect which is which is by looking at the request body. If the attribute exists there, update it.